src/Sarif/Baseline/ResultMatching/DataStructures/ExtractedResult.cs

```diff
@@ -18,13 +18,25 @@ public class ExtractedResult
         public Run OriginalRun { get; set; }
         public string RuleId { get; set; }
 
+        private ReportingDescriptor Rule { get; set; }
```

Comment:
>ReportingDescriptor [](start = 16, length = 19)

I always find it odd when a private member is UpperCased. Because it makes me wonder if the developer intended for it to be public. :) Did you?

Reply:
The coding guidelines I'm familiar with specified that all methods and properties are PascalCased, regardless of access modifier. It's just fields that vary.

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-properties